src/ludamus/templates/panel/import-log.html

```
@@ -0,0 +1,165 @@
+                                            <p class="text-xs text-foreground-muted">
+                                                {% if entry.display_name and entry.title %}{{ entry.display_name }} ·{% endif %}
+                                                {{ entry.attempted_at|date:"DATETIME_FORMAT" }}
+                                            </p>
+                                        </div>
+                                        {% if entry.session_id %}
+                                            <div class="flex flex-wrap items-center gap-2 shrink-0">
+                                                <a class="btn btn-tertiary inline-flex items-center"
+                                                   href="{% url 'panel:proposal-detail' slug=current_event.slug proposal_id=entry.session_id %}">
+                                                    {% translate "View proposal" %}
+                                                </a>
+                                                <form method="post"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)